third_party/nvfuser/csrc/python_frontend/python_bindings.cpp

```diff
@@ -140,18 +140,32 @@ void initNvFuserPythonBindings(PyObject* module) {
           })
       .def(
           "add_output",
-          [](nvfuser::FusionDefinition& self, nvfuser::Scalar output) {
+          [](nvfuser::FusionDefinition& self, nvfuser::Scalar output, c10::optional<nvfuser::Scalar> alias_input = c10::nullopt) {
```

Comment:
Scalars can be aliased in nvfuser, but what is the meaning of that in Python where scalars are immutable? Maybe I am misunderstanding what this represents...